src/win7/winusb_uvc/winusb_uvc.cpp

```diff
@@ -82,6 +82,7 @@ void winusb_uvc_free_device_info(winusb_uvc_device_info_t *info) {
         }
 
         DL_DELETE(info->stream_ifs, stream_if);
+        WinUsb_Free(stream_if->associateHandle);
```

Comment:
must be before DL_DELETE(info->stream_ifs

Reply:
Actually, it should be inside DL_FOREACH_SAFE(info->stream_ifs, stream_if, stream_if_tmp)
so it will free all associatedHandles of all stream_if, no?

Reply:
And also make ->associatehandle NULL